lib/g_searcher/search.ex

```diff
@@ -0,0 +1,18 @@
+defmodule GSearcher.Search do
```

Comment:
how about renaming this context to `SearchResults`? I think `Search` is too generic